lib/default-encoding.js

```diff
@@ -1,8 +1,8 @@
 var defaultEncoding
 /* istanbul ignore next */
-if (process.browser) {
+if (global.process && global.process.browser) {
```

Comment:
```suggestion
if (typeof process !== "undefined" && process.browser) {
```

Reply:
does global not work in webpack 5 ?

Reply:
Good question. I wish I had written more to this suggestion when I wrote it last year.

But if you want to optionally access `process.browser` the proposed code is safe and does not rely on a non-standard `global` variable. [globalThis](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/globalThis) will solve this in a standardized way for modern environments.

Reply:
actually we have a BUNCH of places in here that reference global so if that's something that needs to be removed we should do that in a different pull

Reply:
Sure. But this PR introduces new usages of `global` that can be avoided.